.travis.yml

```diff
@@ -9,7 +11,18 @@ services:
   - docker
 
 script:
-  - make image -e
+  - make image -e IMAGE_BUILD_EXTRA_OPTS="--label quay.expires-after=1h"
```

Comment:
Not sure about this one, but as we build images for e2e tests with `--label quay.expires-after=1h`, in deploy stage we will deploy images with this label and it will expire no? We may need to rebuild images without the label

Reply:
> Not sure about this one, but as we build images for e2e tests with `--label quay.expires-after=1h`, in deploy stage we will deploy images with this label and it will expire no? We may need to rebuild images without the label

Woops, yes this is now badly out-of-sync with the automatic image publishing. Need to fix this (not done, yet)

Thanks for looking at this!

Reply:
I ended up fixing this by just removing `IMAGE_BUILD_EXTRA_OPTS`.

Now, we basically push the `master` image twice, once as part of e2e test and once in the deploy step, but, I don't think that matters as the second push is almost a no-op. I left it like this so that the "official" image deployment would stay the same even if running e2e test was done differently.